### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by sheltyy**
Made a polish translation for demo page, enjoy.

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-  home: Home
-  toggle_dark: Toggle dark mode
-  toggle_langs: Change languages
-intro:
-  desc: Opinionated Vite Starter Template
-  dynamic-route: Demo of dynamic route
-  hi: Hi, {name}!
-  aka: Also known as
-  whats-your-name: What's your name?
-not-found: Not found
```
Comment (hannoeru) [inline locales/en.yml:14]: I think you replaced this file to `pl.yml` but we still need the original `en.yml` file, can you revert this change?

**Review by hannoeru — CHANGES_REQUESTED** [review]

File: locales/en.yml (hunk 1 above)
Comment (sheltyy) [reply locales/en.yml]: Sorry i did it on my phone, so that's why en.yml was accidently removed
